webpack/assets/javascripts/react_app/routes/ForemanRoute.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { layoutPropTypes } from '../components/Layout/Layout';
+import RouteWithLayout from './RouteWithLayout';
+import { noop } from '../common/helpers';
+
+const ForemanRoute = ({
+  render,
+  layout,
+  skipLayout,
```

Comment:
How about passing a falsy value if we do not want a layout? That way you do not need an additional prop:

``js
if (layout) {
// return route with layout
}
// return route without layout
``